include/zcbor_common.h

```diff
@@ -201,6 +206,7 @@ do { \
 #define ZCBOR_ERR_WRONG_RANGE 12
 #define ZCBOR_ERR_ITERATIONS 13
 #define ZCBOR_ERR_ASSERTION 14
+#define ZCBOR_ERR_UNSUPPORTED_TYPE 15
```

Comment:
```suggestion
#define ZCBOR_ERR_UNSUPPORTED 15
```
I think the distinction of `_TYPE` is unneccessary.